src/utils.js

```diff
@@ -1984,10 +1984,12 @@ function replacePackagePlaceHolder (config) {
  * @param {object} action action object
  */
 function validateActionRuntime (action) {
+  // I suspect we have an issue here with 2 kinds of kinds ...
+  // sometimes this method is called with 'sequence' which is a different kind of kind than exec.kind which
+  // comes from action: runtime: in manifest -jm
   if (action.exec && action.exec.kind && action.exec.kind.toLowerCase().startsWith('nodejs:')) {
-    const nodeVer = semver.coerce(action.exec.kind.split(':')[1])
-    if (!semver.satisfies(nodeVer, supportedEngines.node)) {
-      throw new Error(`Unsupported node version in action ${action.name}. Supported versions are ${supportedEngines.node}`)
+    if (SupportedRuntimes.indexOf(action.exec.kind) < 0) {
```

Comment:
can just use array.includes(...) instead which would be more clear

Reply:
no, but I can ! use it 😜